packages/server/src/article/article.service.ts

```diff
@@ -214,4 +223,65 @@ export class ArticleService {
     await this.articleRepository.delete({ id });
     return true;
   }
+
+  async findBookmarkArticles(user: User): Promise<ArticleListInfoDto[]> {
+    const response = [];
+    const bookmarkList = await this.bookmarkRepository.find({
+      where: {
+        user,
+      },
+      relations: [ "article" ],
+    });
+
+    await Promise.all(
+      bookmarkList.map(async (bookmark) => {
+        const { article } = bookmark;
+        const hitCnt = await this.hitRepository.count({ article });
+        const bookmarkCnt = await this.bookmarkRepository.count({ article });
+
+        const formattedDate = moment(article.date).format("YY-DD-MM");
+        response.push(
+          Builder(ArticleListInfoDto)
+            .id(article.id)
+            .boardName(article.board.name)
+            .title(article.title)
+            .date(formattedDate)
+            .hits(hitCnt)
+            .scraps(bookmarkCnt)
+            .build(),
+        );
+      }),
+    );
+
+    return response;
+  }
+
+  async findSubscribeArticles(user: User): Promise<ArticleListInfoDto[]> {
+    const response = [];
+    const boardIdList = await this.subscribeService.findBoardByUser(user.id);
+    const articleList = await this.articleRepository.findRecentArticlesByBoard(
+      boardIdList,
+    );
+
+    await Promise.all(
+      articleList.map(async (article) => {
+        const hitCnt = await this.hitRepository.count({ article });
+        const bookmarkCnt = await this.bookmarkRepository.count({ article });
+
+        const formattedDate = moment(article.date).format("YY-DD-MM");
+        response.push(
+          Builder(ArticleListInfoDto)
+            .id(article.id)
+            .boardName(article.board.name)
+            .title(article.title)
+            .date(formattedDate)
+            .hits(hitCnt)
+            .scraps(bookmarkCnt)
+            .build(),
+        );
+      }),
+    );
```

Comment:
```suggestion
    response = await Promise.all(
      articleList.map(async (article) => {
        const hitCnt = await this.hitRepository.count({ article });
        const bookmarkCnt = await this.bookmarkRepository.count({ article });

        const formattedDate = moment(article.date).format("YY-DD-MM");
        
        return Builder(ArticleListInfoDto)
            .id(article.id)
            .boardName(article.board.name)
            .title(article.title)
            .date(formattedDate)
            .hits(hitCnt)
            .scraps(bookmarkCnt)
            .build()
   
      }),
    );
```

map은 배열을 받아서 새로운 배열을 반환하는 함수이기에 안에서 `push`하는거보다는 `return`하고 반환값을 사용하는게 적합한 사용법입니다.

Reply:
> 고생많으셨습니다~~! 👏👏



> 고생많으셨습니다~~! 👏👏

moment, map 부분은 리팩토링 브랜치 따로 파서 한번에 수정하겠습니다! 
조언 감사합니다~! 🐳